### ISSUE TITLE
how to use examples/asynchronous_publisher_example.py?

### ISSUE THREAD

**Issue by beyondasce** [pika/pika#1075]
Do you need to create an ExamplePublisher object each time you want to push multiple data? How to use a channel to pass multiple data?

**Comment by lukebakken** [comment]
Hello!

In the future, please use the [`pika-python`](https://groups.google.com/forum/#!forum/pika-python) for questions like these. Another option is to use [`rabbitmq-users`](https://groups.google.com/forum/#!forum/rabbitmq-users) - the same RabbitMQ core team members try to keep up with both.

You need to customize the example code to do what you'd like. Creating an `ExamplePublisher` class for each message would be _extremely_ inefficient because it would create a new connection and channel for every message.

Basically, you need to implement your own means to publish messages -
https://github.com/pika/pika/blob/0.11.2/examples/asynchronous_publisher_example.py#L259-L289

Feel free to follow up on one of those mailing lists if you have questions.

**Comment by beyondasce** [comment]
Ok,thank you.
